### PR TITLE
Deprecate the package from PyPI

### PR DESCRIPTION
**Pull request by paddyroddy**
Following steps in #4, and idea from https://www.dampfkraft.com/code/how-to-deprecate-a-pypi-package.html. Once this is merged we'll need to tag to get the action to run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,3 @@
...
-[GNU LGPL v3.0]: http://www.gnu.org/licenses/lgpl-3.0.txt
-[Apache Software License 2.0]: http://www.apache.org/licenses/LICENSE-2.0
-[Mozilla Public License 2.0]: https://www.mozilla.org/media/MPL/2.0/index.txt
-[cookiecutter-napari-plugin]: https://github.com/napari/cookiecutter-napari-plugin
-[file an issue]: https://github.com/quantumjot/napari-btrack-reader/issues
-[napari]: https://github.com/napari/napari
-[tox]: https://tox.readthedocs.io/en/latest/
-[pip]: https://pypi.org/project/pip/
-[PyPI]: https://pypi.org/
+The functionality of this package has been moved to https://github.com/quantumjot/BayesianTracker.
```
Comment (p-j-smith) [inline README.md]: Maybe change to :

The functionality of this package has been integrated into [BayesianTracker](https://github.com/quantumjot/BayesianTracker).

**Review by p-j-smith — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paddyroddy) [reply README.md]: Done. Although was trying to avoid the name in the hope that it gets renamed to `btrack`.

**Review by p-j-smith — APPROVED** [review]

**Comment by p-j-smith** [comment]
lgtm

**Comment by paddyroddy** [comment]
Tag pushed

**Pull request merged**